src/Backend/Modules/Location/Engine/Model.php

```diff
@@ -217,6 +217,7 @@ public static function insert(array $item): int
         // insert extra
         $item['extra_id'] = BackendModel::insertExtra(
             ModuleExtraType::widget(),
+            'Location',
```

Comment:
The Location module has an action called Index so I'm guessing this is the correct default. This means I can't change the default, so I had to change it here instead